projects/kit/pipes/field-error/field-error-pipe.ts

```diff
@@ -0,0 +1,159 @@
+import {Inject, Optional, Pipe, PipeTransform, Self} from '@angular/core';
+import {
+    AbstractControl,
+    FormArrayName,
+    FormGroupDirective,
+    FormGroupName,
+    NgControl,
+} from '@angular/forms';
+import {tuiPure, TuiValidationError} from '@taiga-ui/cdk';
+import {TUI_VALIDATION_ERRORS} from '@taiga-ui/kit';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+import {EMPTY, isObservable, merge, Observable, of} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+const EMPTY_RECORD = {};
+
+@Pipe({
+    name: 'tuiFieldError',
+    pure: false,
+})
+export class TuiFieldErrorPipe implements PipeTransform {
+    private order: ReadonlyArray<string> = [];
+
+    constructor(
+        @Optional()
+        @Self()
+        @Inject(NgControl)
+        private readonly ngControl: NgControl | null,
+        @Optional()
+        @Self()
+        @Inject(FormArrayName)
+        private readonly formArrayName: FormArrayName | null,
+        @Optional()
+        @Self()
+        @Inject(FormGroupName)
+        private readonly formGroupName: FormGroupName | null,
+        @Optional()
+        @Self()
+        @Inject(FormGroupDirective)
+        private readonly formGroup: FormGroupDirective | null,
+        @Inject(TUI_VALIDATION_ERRORS)
+        private readonly validationErrors: Record<
+            string,
+            PolymorpheusContent | Observable<PolymorpheusContent>
+        >,
+    ) {
+        if (this.ngControl && !this.ngControl.valueAccessor) {
+            this.ngControl.valueAccessor = this;
+        }
```

Comment:
i'm not really sure how good it is to override value accessor by instance pipe)) but I hope it is necessary)

Reply:
It's ok since this instance of `NgControl` is only for error management.